c7n/resources/s3.py

```diff
@@ -2595,16 +2600,20 @@ def get_event_buckets(self, session, trails):
 
     def process(self, resources, event=None):
         trails = self.manager.get_resource_manager('cloudtrail').resources()
+        local_trails = self.filter_resources(
+            trails,
+            "split(':', TrailARN)[4]", (self.manager.account_id,)
```

Comment:
interesting use of expression

Reply:
I guess is organization trail doesn't work if its in the same account re simpler expression.